app/src/components/widgets/singleLevelWidgets/tables/flakyTests/flakyTests.jsx

```diff
@@ -19,8 +19,22 @@ import PropTypes from 'prop-types';
 import { TestsTableWidget } from '../components/testsTableWidget';
 import * as cfg from './flakyTestsCfg';
 
+const prepareWidgetData = ({ flaky }) =>
+  flaky.map((item) => ({
+    flakyCount: item.flakyCount,
+    itemName: item.itemName,
+    total: item.total,
+    uniqueId: item.uniqueId,
+    startTime: item.startTime,
+    statuses: item.statuses.reverse(),
```

Comment:
It seems to me, it would be better to use spread operator here: `{...item, statuses: item.statuses.revese()}`